COMETwebapp/wwwroot/Scripts/babylonInterop.js

```diff
@@ -189,9 +189,15 @@ function AddPrimitive(primitive, color) {
         case "Cylinder":        CreateCylinder(primitive, color); break;

```

Comment:
When you have a lot of cases, using a Dictionary<string, Action or Func<>> works faster.